wasmcloud_host/README.md

```diff
@@ -1,21 +1,146 @@
 # wasmCloud Host - Web UI Dashboard
+
 This is the web UI dashboard that provides for a basic way to interact with a host and its associated lattice. This web application automatically starts the [host_core](../host_core/README.md) application as a dependency.
 
-## Starting the Host and Web UI Dashboard
+To learn more about wasmCloud, please view the [Documentation](https://wasmcloud.dev).
+
+
+## Building the host
+
+Clone the repo `wasmCloud/wasmcloud-otp` into some suitable development directory
+
+```console
+git clone https://github.com/wasmCloud/wasmcloud-otp.git
+```
+
+
+### Language Prerequisites
+
+Install the latest version of [Rust](https://www.rust-lang.org/tools/install) and [Elixir](https://elixir-lang.org/install.html).
+Elixir requires the Erlang VM to be installed, but many of the installation methods listed will take care of this for you.
+
+If building the `httpserver` provider from source, you'll also need [Go](https://golang.org/doc/install)
```

Comment:
I think this is not the real `httpserver` provider. That's in the `capability-providers` repo. So I wouldn't mention it at all